src/main/java/edu/ucsd/sbrg/bigg/SBMLPolisher.java

```diff
@@ -50,8 +50,8 @@
 import de.zbit.kegg.AtomBalanceCheck.AtomCheckResult;
 import de.zbit.util.progressbar.AbstractProgressBar;
 import de.zbit.util.progressbar.ProgressBar;
-import edu.ucsd.sbrg.util.SBMLFix;
-import edu.ucsd.sbrg.util.SBMLUtils;
+import main.java.edu.ucsd.sbrg.util.SBMLFix;
+import main.java.edu.ucsd.sbrg.util.SBMLUtils;
```

Comment:
This is not correct. Moving the files from `/src/` to `/src/main/java` doesn't mean we have to change all the packages. Please change this again. The package is still `edu.ucsd.sbrg.util...`. The source code is now just in a different folder.